gm_dotnet_managed/GmodNET/Startup.cs

```diff
@@ -10,10 +10,7 @@
 
 namespace GmodNET
 {
-    // This delegate is needed for native interop and its signature nust match Startup.Main's one.
-    internal delegate IntPtr MainDelegate(IntPtr lua_base, IntPtr native_version_string, int version_string_length, IntPtr param);
```

Comment:
Was this deprecated/unused? Or has it been replaced with something?

Reply:
From the C++ code, am I right in saying we just call Main directly now instead?

Reply:
You are right: previous versions of hostfxr API required a delegate type declared to make a call to managed function. It was needed for various reasons, like specifying native calling conventions. With .NET 5 and it's new `UnmanagedCallersOnly` attribute we no longer need this declaring delegates at all.